nautobot_chatops_ipfabric/ipfabric.py

```diff
@@ -41,6 +47,13 @@ def get_devices_info(self, snapshot_id="$last", limit=DEFAULT_PAGE_LIMIT):
         }
         return self.get_response("/api/v1/tables/inventory/devices", payload)
 
+    def get_os_version(self):
+        """Return Device info."""
+        logger.debug("Received OS version request")
+
+        payload = {}
+        return self.get_response_json("GET", "/api/v1/os/version", payload)
```

Comment:
why not doing all the work here?
```suggestion
        response = self.get_response_json("GET", "/api/v1/os/version", payload)
        return float(response.get("version", "0.0").rpartition(".")[0])
```

Reply:
yeah, could do that